### PR TITLE
Include event to AlarmMessage.

### PR DESCRIPTION
**Pull request by chenmudu**

### CODE REVIEW

File: alarm.graphqls [hunk 1]
```diff
@@ -37,6 +38,32 @@ input AlarmTag {
     value: String
 }
 
+type Event {
```
Comment (wu-sheng) [inline alarm.graphqls:41]: Please share the object in `event.graphqls`. You don't need this duplicate one.

File: alarm.graphqls [hunk 2]
```diff
@@ -24,6 +24,7 @@ type AlarmMessage {
     scope: Scope
     id: ID!
     message: String!
+    events: [Event]!
```
Comment (wu-sheng) [inline alarm.graphqls:27]:
```suggestion
    events: [Event!]!
```

File: alarm.graphqls (hunk 1 above)
Comment (chenmudu) [reply alarm.graphqls:41]: Can I apply the `Event` of event.graphqls directly to alarm.graphqls?  Sorry, I'm not familiar with this query protocol.
Like this? 
image

Comment (wu-sheng) [reply alarm.graphqls:41]: Yes, you could use it directly. And more importantly, you should try the latest version in your OAP, and make sure it is suitable(compilable and runnable). There are a lot of verifications when you boot the OAP server.

File: alarm.graphqls (hunk 2 above)
Comment (chenmudu) [reply alarm.graphqls:27]: Ok. my earlier fears were misplaced.

File: alarm.graphqls (hunk 1 above)
Comment (chenmudu) [reply alarm.graphqls:41]: OK, Thanks a lot.

File: alarm.graphqls [hunk 3]
```diff
@@ -37,6 +38,7 @@ input AlarmTag {
     value: String
 }
 
+
```
Comment (wu-sheng) [inline alarm.graphqls:41]:
```suggestion
```

Be careful, don't change an unnecessary thing, due to not confusing others.

**Comment by wu-sheng** [comment]
And with this change, you need another new issue to track UI update.

File: alarm.graphqls (hunk 3 above)
Comment (chenmudu) [reply alarm.graphqls]: > Be careful, don't change an unnecessary thing, due to not confusing others.

Yeah, I'm so sorry. I'll keep it in mind.

Comment (chenmudu) [reply alarm.graphqls]: Yeah, Thanks a lot.

**Comment by chenmudu** [comment]
> And with this change, you need another new issue to track UI update.

What should I do, Create a PR in the main repository to track UI updates or do I need to keep an eye on related UI updates?

**Comment by wu-sheng** [comment]
> > And with this change, you need another new issue to track UI update.
> 
> What should I do, Create a issue in the main repository to track UI updates or do I need to keep an eye on related UI updates? 😀

Submit another issue, talk about what you are going to change, UI should visualize this too.

**Comment by chenmudu** [comment]
> > > And with this change, you need another new issue to track UI update.
> > 
> > 
> > What should I do, Create a issue in the main repository to track UI updates or do I need to keep an eye on related UI updates? 😀
> 
> Submit another issue, talk about what you are going to change, UI should visualize this too.

Copy that. I'll describe it clearly. Thanks.

**Comment by wu-sheng** [comment]
After merge, you need to update the submodule in your branch about query-receiver.

**Comment by chenmudu** [comment]
> After merge, you need to update the submodule in your branch about query-receiver.

Copy.

**Pull request merged**